relay-server/src/actors/global_config.rs

```diff
@@ -0,0 +1,121 @@
+use std::sync::Arc;
+use std::time::Duration;
+
+use relay_dynamic_config::GlobalConfig;
+use relay_system::{Addr, AsyncResponse, FromMessage, Interface, Sender, Service};
+use tokio::sync::mpsc::{self, UnboundedSender};
+
+use crate::actors::processor::EnvelopeProcessor;
+use crate::actors::project_upstream::GetProjectStates;
+use crate::actors::upstream::{SendQuery, UpstreamRelay};
+
+/// Service implementing the [`GlobalConfiguration`] interface.
+///
+/// The service is responsible for fetching the global config and
+/// forwarding it to the services that require it, and for serving downstream relays.
+#[derive(Debug)]
+pub struct GlobalConfigurationService {
+    global_config: Arc<GlobalConfig>,
+    envelope_processor: Addr<EnvelopeProcessor>,
+    upstream: Addr<UpstreamRelay>,
+}
+
+/// Service interface for the [`GetGlobalConfig`] message.
+pub enum GlobalConfiguration {
+    GetGlobalConfig(Sender<Arc<GlobalConfig>>),
+}
```

Comment:
It's a good practice we have to add more documentation to the interface describing what it does and how to use it.
This can include the workflow and the messages it receives or sends to update its state.